Lavinia-api/Data/Countries/NO/PE/Elections.csv

```diff
@@ -1,4 +1,20 @@
-﻿Year;Algorithm;FirstDivisor;Threshold;Seats;LevelingSeats
-2017;Sainte Laguës (modified);1.4;4.0;150;19
-2013;Sainte Laguës (modified);1.4;4.0;150;19
-2009;Sainte Laguës (modified);1.4;4.0;150;19
\ No newline at end of file
+﻿Year;Algorithm;FirstDivisor;Threshold;AreaFactor;Seats;LevelingSeats
+2017;Sainte Laguës (modified);1.4;4.0;1.8;150;19
+2013;Sainte Laguës (modified);1.4;4.0;1.8;150;19
+2009;Sainte Laguës (modified);1.4;4.0;1.8;150;19
+2005;Sainte Laguës (modified);1.4;4.0;1.8;150;19
+2001;Sainte Laguës (modified);1.4;4.0;1.8;157;8
+1997;Sainte Laguës (modified);1.4;4.0;1.8;157;8
+1993;Sainte Laguës (modified);1.4;4.0;1.8;157;8
+1989;Sainte Laguës (modified);1.4;4.0;1.8;157;8
+1985;Sainte Laguës (modified);1.4;0.0;-1;157;0
+1981;Sainte Laguës (modified);1.4;0.0;-1;155;0
+1977;Sainte Laguës (modified);1.4;0.0;-1;155;0
+1973;Sainte Laguës (modified);1.4;0.0;-1;156;0
+1969;Sainte Laguës (modified);1.4;0.0;-1;150;0
+1965;Sainte Laguës (modified);1.4;0.0;-1;150;0
+1961;Sainte Laguës (modified);1.4;0.0;-1;150;0
+1957;Sainte Laguës (modified);1.4;0.0;-1;150;0
+1953;Sainte Laguës (modified);1.4;0.0;-1;150;0
+#1949;d'Hondt;1.0;0.0;-1;112;0
```

Comment:
Should we really be commenting this out instead of removing it? I seem to recall we had a habit of deleting "dead code" and using git to revisit it if needed.

As you are more familiar than me with the current state of the API, I think you should create an issue to resolve this "D'Hondt" problem explaining why it doesn't work and what will be required to fix it.